datasets/kelm/dataset_infos.json

```diff
@@ -0,0 +1 @@
+{"default": {"description": "Data-To-Text Generation involves converting knowledge graph (KG) triples of the form (subject, relation, object) into\na natural language sentence(s). This dataset consists of English KG data converted into paired natural language text.\nThe generated corpus consists of \u223c18M sentences spanning \u223c45M triples with \u223c1500 distinct relations.\n", "citation": "@misc{agarwal2020large,\n      title={Large Scale Knowledge Graph Based Synthetic Corpus Generation for Knowledge-Enhanced Language Model Pre-training}, \n      author={Oshin Agarwal and Heming Ge and Siamak Shakeri and Rami Al-Rfou},\n      year={2020},\n      eprint={2010.12688},\n      archivePrefix={arXiv},\n      primaryClass={cs.CL}\n}\n", "homepage": "https://github.com/google-research-datasets/KELM-corpus", "license": "", "features": {"triple": {"dtype": "string", "id": null, "_type": "Value"}, "sentence": {"dtype": "string", "id": null, "_type": "Value"}}, "post_processed": null, "supervised_keys": null, "builder_name": "ag_news", "config_name": "default", "version": {"version_str": "0.0.0", "description": null, "major": 0, "minor": 0, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 1343187306, "num_examples": 6371131, "dataset_name": "ag_news"}, "validation": {"name": "validation", "num_bytes": 167790917, "num_examples": 796471, "dataset_name": "ag_news"}, "test": {"name": "test", "num_bytes": 167921750, "num_examples": 796493, "dataset_name": "ag_news"}}, "download_checksums": {"https://storage.googleapis.com/gresearch/kelm-corpus/quadruples-train.tsv": {"num_bytes": 1305075939, "checksum": "55c1d4e1beaccda979fc7193e192bc48af05bf3357bd7c14b93ba750fca91c55"}, "https://storage.googleapis.com/gresearch/kelm-corpus/quadruples-validation.tsv": {"num_bytes": 163026560, "checksum": "802c26a7856b16f09e5380e54f115bee66d83539cc2f41bb39fbf651b99a31ed"}, "https://storage.googleapis.com/gresearch/kelm-corpus/quadruples-test.tsv": {"num_bytes": 163157370, "checksum": "d41be1cb6feed48d938136b8783ba67a9bfc262fc0614df6208937381de11e36"}}, "download_size": 1631259869, "post_processing_size": null, "dataset_size": 1678899973, "size_in_bytes": 3310159842}}
```

Comment:
can you regenerate this file ?
something is wrong in it (ex: `"builder_name": "ag_news"`)

Reply:
Haha dang I'm 2 for 2 with that kind of error. At least I didn't merge this one 🤦‍♂️ fixed.